src/HonzaBotner.Discord.Services/EventHandlers/ReminderReactionsHandler.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Threading.Tasks;
+using DSharpPlus.Entities;
+using DSharpPlus.EventArgs;
+using HonzaBotner.Discord.EventHandler;
+using HonzaBotner.Discord.Services.Options;
+using HonzaBotner.Services.Contract;
+using Microsoft.Extensions.Options;
+
+namespace HonzaBotner.Discord.Services.EventHandlers
+{
+    public class ReminderReactionsHandler : IEventHandler<MessageReactionAddEventArgs>
+    {
+        private readonly ReminderOptions _options;
+
+        private readonly IRemindersService _service;
+
+        public ReminderReactionsHandler(IOptions<ReminderOptions> options, IRemindersService service)
+        {
+            _options = options.Value;
+            _service = service;
+        }
+
+        public async Task<EventHandlerResult> Handle(MessageReactionAddEventArgs arguments)
+        {
+            // TODO: Move this to framework level code
+            if (arguments.User.IsBot)
+            {
+               return EventHandlerResult.Stop;
+            }
+
+            var emoji = arguments.Emoji;
```

Comment:
Performance: shouldn't we check first, if the emoji is in valid range first? So we don't query the database unless it is necessary.